draft-ietf-jsonpath-base.md

```diff
@@ -1066,9 +1066,11 @@ regex-expr   = regex-op regex
 regex-op     = "=~"                                   ; regular expression match
 regex        = <TO BE DEFINED>
 
-contain-expr = in-op container
+contain-expr = containable in-op container
+containable  = rel-path / json-path /                 ; path to primitive value 
+               number / string-literal
 in-op        = " in "                                 ; in operator
```

Comment:
Not sure why spaces need to be included in this token. Wouldn't it be better to require arbitrary whitespace either side of the token instead?

Reply:
In fact I was thinking about using whitespace `W` here. But there are some other locations in the spec where this should be done ... perhaps within a another special whitespace PR.

Reply:
Yes, a whitespace cleanup is probably overdue.